src/test/java/lotto/LottoTicketTest.java

```diff
@@ -36,9 +42,30 @@ void checkDuplicate() {
     @DisplayName("발급받은 로또번호 중 일치여부를 판단해 일치하는 개수를 판단한다")
     @Test
     void matchesLottoTicket() {
-        LottoTicket lottoTicket = LottoTicket.of(List.of(1, 2, 3, 4, 5, 45));
+        LottoTicket secondRankLottoTicket = LottoTicket.of(List.of(1, 2, 3, 4, 5, 45));
         List<Integer> winningLottoTicket = List.of(1, 2, 3, 4, 5, 6);
-        int winningCount = lottoTicket.matchesLottoNumberCount(winningLottoTicket);
+        int winningCount = secondRankLottoTicket.matchesLottoNumberCount(winningLottoTicket);
         assertThat(winningCount).isEqualTo(5);
     }
+
+    @DisplayName("당첨번호와 보너스번호에 따른 등수를 구한다")
+    @ParameterizedTest
+    @MethodSource("lottoRank")
+    void prize(List<Integer> winningLottoTicket, LottoTicket lottoTicket, LottoNumber bonusNumber, LottoPrize expectedLottoPrize) {
+        LottoPrize lottoPrize = lottoTicket.getWinningPrize(winningLottoTicket, bonusNumber);
+        assertThat(lottoPrize).isEqualTo(expectedLottoPrize);
+    }
```

Comment:
```suggestion
    @DisplayName("당첨번호 중 5자리가 같고 보너스 번호를 포함하고 있다면, 2등") 
    @Test
    void rank2() {
        List<Integer> winning = List.of(1, 2, 3, 4, 5, 6);
        LottoNumber bonus = new LottoNumber(45);

        LottoTicket ticket = LottoTicketFixture.of(1, 2, 3, 4, 5, 45);
        LottoPrize actual = ticket.rank(winning, bonus);
        assertThat(actual).isEqualTo(RANK2);
    }
```

- 테스트를 직관적으로 표현하고 싶다면, 너무 추상화시키기 보다는 구현을 드러내는 것도 좋은 방법입니다. 그리고 테스트하고자 하는 바를 DisplayName으로 표현해주니 더 이해하기가 수월합니다.
- 테스트는 given, when, then이 명확하게 드러날 때 좀 더 직관적이라고 생각합니다.
- 생성하는 로직은 Fixture 등으로 추상화하여 테스트하고자하는 대상을 부각시키는 게 좋다고 생각합니다.